pixeltable/exec/expr_eval_node.py

```diff
@@ -50,7 +50,14 @@ def __next__(self) -> DataRowBatch:
 
     def _open(self) -> None:
         warnings.simplefilter("ignore", category=TqdmWarning)
-        if self.ctx.show_pbar:
+        # This is a temporary hack. When indexable strings were introduced via new (anonymous) computed columns,
```

Comment:
I wasn't sure what this is referring to and had to look it up.

Better: "When B-tree indices on string columns were implemented (via computed columns that invoke the BtreeIndex.str_filter udf), it resulted...
This special-cases the str_filter call to suppress..."